app/views/patients/index.html.slim

```diff
@@ -5,28 +5,23 @@
     / This search field doesn't work yet
     form.form-inline.pull-right
       .form-group
-        / Well this is embarrasing, TODO (emilyville)
-        | &nbsp;
-        input#exampleInputName2.form-control placeholder=("Patient ID or name") name="query" type="text" value="#{@query}" /
-        | &nbsp;
+        input#exampleInputName2.form-control placeholder=("Name or patient ID") last_name="query" type="text" value="#{@query}" /
```

Comment:
This search field does no searching (just the UI that's there now), so I'm not too concerned with it.

I think @huttj offered to address the search box at some point along with the reports search. Might make sense to punt until then?